apps/api/src/domain/models/bibliographic-reference/entities/JournalArticleBibliographicReferenceData.ts

```diff
@@ -0,0 +1,76 @@
+import {
+    ArrayNotEmpty,
+    IsOptional,
+    IsStringWithNonzeroLength,
+    IsUrl,
+} from '../../../../../../../libs/validation/src';
+import { DTO } from '../../../../types/DTO';
+import { isNullOrUndefined } from '../../../utilities/validation/is-null-or-undefined';
+import BaseDomainModel from '../../BaseDomainModel';
+import { IBibliographicReferenceData } from '../interfaces/IBibliographicReferenceData';
+import { BibliographicReferenceType } from '../types/BibliographicReferenceType';
+import Creator from '../types/Creator';
+
+export default class JournalArticleBibliographicReferenceData
+    extends BaseDomainModel
+    implements IBibliographicReferenceData
+{
+    readonly type = BibliographicReferenceType.journalArticle;
+
+    @IsStringWithNonzeroLength()
+    readonly title: string;
+
+    @ArrayNotEmpty()
+    readonly creators: Creator[];
```

Comment:
The class should be called `ZoteroCreator` or `BibliographicReferenceCreator` or something like that. `Creator` is too generic.